src/browser/services/fetch-client.js

```diff
@@ -2,8 +2,10 @@ import { Client, createError } from '../../core';
 import isHtml from 'is-html';
 
 export default class FetchClient extends Client {
-  request(url, method = 'get', body, headers = {}) {
-    const options = {
+  request(url, body, headers = {}, options = {}) {
+    const { method = 'get' } = options;
+    const requestOptions = {
+      ...options,
```

Comment:
Are you destructuring the `options` object before the declaration of the other properties to prevent anyone from overriding those specific properties?

Reply:
Yes!

Reply:
Very well. 👍